actioncable/blade.yml

```diff
@@ -5,3 +5,9 @@ load_paths:
 
 logical_paths:
   - test.js
+
+build:
+  logical_paths:
+    - action_cable.js
+  path: lib/assets/compiled
+  js_compressor: uglifier
```

Comment:
I think I'd prefer to keep this in a form amenable to hackery -- I know it's Bad, but I'll often open a file in an installed gem to install a temporary `puts`; retaining the ability to do the same with `console.log` seems desirable... especially as it doesn't really matter -- it'll be compressed eventually either way.

(Bonus argument: if we pre-compress, sprockets + source-maps will leave this file unpleasant to debug even when everything else is nice and shiny.)